$OpenBSD: patch-src_CImg_h,v 1.2 2021/05/16 09:47:28 espie Exp $

quick hack to allow parallel to work since we don't have a standard include
guard
Index: src/CImg.h
--- src/CImg.h.orig
+++ src/CImg.h
@@ -390,6 +390,9 @@ enum {FALSE_WIN = 0};
 #include <X11/Xutil.h>
 #include <X11/keysym.h>
 #include <pthread.h>
+#ifndef _PTHREAD_H
+#  define _PTHREAD_H
+#endif
 #ifdef cimg_use_xshm
 #include <sys/ipc.h>
 #include <sys/shm.h>
